src/generators.h

```diff
@@ -18,14 +18,17 @@ class GeneratorInterface {
     return deltas;
   }
 
-  virtual auto CreateDeltas(uint32_t t0, uint32_t count) const -> std::vector<double> = 0;
+  [[nodiscard]] virtual auto CreateDeltas(uint32_t t0, uint32_t count) const
+      -> std::vector<double> = 0;
   virtual ~GeneratorInterface() = default;
 };
 
 class ExponentialGenerator : public GeneratorInterface {
  public:
   ExponentialGenerator(double gamma1, double gamma2) : gamma1_(gamma1), gamma2_(gamma2) {}
-  auto CreateDeltas(uint32_t t0, uint32_t count) const -> std::vector<double> override {
+  // Creates deltas as defined in COR01.pdf.
+  [[nodiscard]] auto CreateDeltas(uint32_t t0, uint32_t count) const
+      -> std::vector<double> override {
     t0 = std::min(t0, count - 1);
```

Comment:
I have question regarding this line. Is count always bigger than 0?

Reply:
Yeah, It's the length in days.